Makefile

```diff
@@ -1,7 +1,28 @@
-.PHONY: lint
+VENV=venv
+PIP_LOG=pip-log.txt
+PRECOMMIT_SRC=.githooks/pre-commit
+PRECOMMIT_DST=.git/hooks/pre-commit
+
+export VIRTUAL_ENV := $(abspath ${VENV})
+export PATH := ${VIRTUAL_ENV}/bin:${PATH}
```

Comment:
I'd rather not mess with the PATH and use the full `${VENV}/bin/foo` every time. Feels less magic and there are less chances of messing up the system python packages by mistake.

Just my opinion though

Reply:
Fair enough. It'll also make it possible to copy-paste commands output by `make`.